msmbuilder/featurizer/multichain.py

```diff
@@ -0,0 +1,510 @@
+# Author: Brooke Husic <brookehusic@gmail.com>
+# Contributors: 
+# Copyright (c) 2016, Stanford University and the Authors
+# All rights reserved.
+
+from __future__ import print_function, division, absolute_import
+
+import warnings
+
+import mdtraj as md
+from mdtraj.utils import ensure_type
+from mdtraj.utils.six import string_types
+from mdtraj.utils.six.moves import xrange
+import numpy as np
+import itertools
+
+from msmbuilder import libdistance
+from . import Featurizer, zippy_maker, dict_maker, featurize_all
+
+
+class LigandFeaturizer(Featurizer):
+    """Base class for objects that featurize protein-ligand trajectories.
+
+    Notes
+    -----
+    At the bare minimum, a featurizer must implement the `partial_transform(traj)`
+    member function.  A `transform(traj_list)` for featurizing multiple
+    trajectories in batch will be provided.
+    """
+
+
+    def __init__(self, protein_chain='auto', ligand_chain='auto',
+                 reference_frame=None):
+        self.protein_chain = protein_chain
+        self.ligand_chain = ligand_chain
+        self.reference_frame = reference_frame
+
+        if reference_frame is None:
+            raise ValueError("Please specify a reference frame")
```

Comment:
This is based on how [`RMSDFeaturizer`](https://github.com/msmbuilder/msmbuilder/blob/master/msmbuilder/featurizer/featurizer.py#L271-L278) does things (I know this is the base class and is a different case, but I thought it might be too risky to make a default). I could change it to the topology though? Although aren't there some trajectory formats that don't require a top?